packages/core/src/debug/debug_node.ts

```diff
@@ -257,7 +257,14 @@ class DebugElement__POST_R3__ extends DebugNode__POST_R3__ implements DebugEleme
 
     const properties = collectPropertyBindings(tNode, lView, tData);
     const hostProperties = collectHostPropertyBindings(tNode, lView, tData);
-    return {...properties, ...hostProperties};
+    const output = {...properties, ...hostProperties};
+    const classes = this.classes;
+    for (const className of Object.keys(classes)) {
+      if (classes[className]) {
+        output.className = output.className ? output.className + ` ${className}` : className;
```

Comment:
I think this needs to be bracket access for Closure:

```suggestion
        output['className'] = output['className'] ? output['className'] + ` ${className}` : className;
```